src/main/java/nl/tudelft/ti2206/group9/shop/skin/AbstractSkin.java

```diff
@@ -0,0 +1,61 @@
+package nl.tudelft.ti2206.group9.shop.skin;
+
+import javafx.scene.paint.PhongMaterial;
+import nl.tudelft.ti2206.group9.gui.Style;
+import nl.tudelft.ti2206.group9.shop.ShopItem;
+
+/**
+ * The abstract shop item class for skins, based on the requirements from the
+ * ShopItem interface. Besides, every skin item has it's own texture.
+ * @author Maikel and Mitchell.
+ */
+@SuppressWarnings("restriction") // SuppresWarnings, because
+// PhongMaterials are needed for the textures.
+public abstract class AbstractSkin implements ShopItem {
```

Comment:
This class (and AbstractSoundtrack) are abstract, but do not have any abstract methods. The SAT don't complain about this? I don't see any warnings, so I guess it's fine. Furthermore, I agree that the superclass shouldn't be able to be instantiated.

<review, logic